_docs_integrate/error-codes.md

```diff
@@ -24,74 +24,163 @@ required_by:
+| {% include anchor a="error.transport.files.maxFileSizeExceeded" %}                                                | The given file content size exceeds the max file size the backbone accepts.                                                                                                                                                                                     |
+| {% include anchor a="error.transport.files.plaintextHashMismatch" %}                                              | The actual hash of the plaintext does not match the given plaintextHash. Something went wrong while encrypting/decrypting the file.                                                                                                                             |
+| {% include anchor a="error.transport.general.baseUrlNotSet" %}                                                    | The baseUrl was not set. ([Configuration options]({% link _docs_operate/configuration.md %}#configuration-options))                                                                                                                                             |
+| {% include anchor a="error.transport.general.platformClientIdNotSet" %}                                           | The platform clientId was not set. ([Configuration options]({% link _docs_operate/configuration.md %}#configuration-options))                                                                                                                                   |
+| {% include anchor a="error.transport.general.platformClientInvalid" %}                                            | The combination of platform clientId and clientSecret is invalid. ([Configuration options]({% link _docs_operate/configuration.md %}#configuration-options))                                                                                                    |
+| {% include anchor a="error.transport.general.platformClientSecretNotSet" %}                                       | The platform clientSecret was not set. ([Configuration options]({% link _docs_operate/configuration.md %}#configuration-options))                                                                                                                               |
+| {% include anchor a="error.transport.identity.realmLength" %}                                                     | The given realm is not 3 characters long.                                                                                                                                                                                                                       |
```

Comment:
I think we deleted realms with v5, so you can delete this error here as well.

Reply:
I think, when we will merge V5 we should do another look over all error codes changes.